File: AndroidSlideToUnlock/app/src/main/java/com/example/shakil/androidslidetounlock/MainActivity.java
package com.example.shakil.androidslidetounlock;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;

import com.ncorti.slidetoact.SlideToActView;

public class MainActivity extends AppCompatActivity {

    SlideToActView slideToActView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        slideToActView = findViewById(R.id.slide_to_unlock);
        slideToActView.setOnSlideCompleteListener(new SlideToActView.OnSlideCompleteListener() {
            @Override
            public void onSlideComplete(SlideToActView slideToActView) {
                startActivity(new Intent(MainActivity.this, HomeActivity.class));
            }
        });
    }

    @Override
    protected void onResume() {
        super.onResume();
        slideToActView.resetSlider();
    }
}

File: KotlinSlideToUnlock/app/src/main/java/com/example/shakil/kotlinslidetounlock/MainActivity.kt
package com.example.shakil.kotlinslidetounlock

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.ncorti.slidetoact.SlideToActView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        slide_to_unlock.onSlideCompleteListener = object :SlideToActView.OnSlideCompleteListener{
            override fun onSlideComplete(view: SlideToActView) {
                startActivity(Intent(this@MainActivity, HomeActivity::class.java))
            }
        }
    }

    override fun onResume() {
        super.onResume()
        slide_to_unlock.resetSlider()
    }
}
